src/components/lib/ConnectWalletDialog.js

```diff
@@ -1,105 +1,198 @@
 import React, { useState } from 'react'
 import { useWeb3React } from '@web3-react/core'
 import { InjectedConnector } from '@web3-react/injected-connector'
+import { LedgerConnector } from '../../connectors/ledger'
+import { TrezorConnector } from '../../connectors/trezor'
 
-const SUPPORTED_CHAIN_IDS = [
-	// Mainnet
-	1,
-	// Ropsten
-	3,
-	// Rinkeby
-	4,
-	// Dev chains (Ganache, Geth)
-	123, // Low chainId to workaround ledgerjs signing issues.
-	1337,
-	// Keep testnet
-	1101
-]
+const CHAIN_ID = process.env.CHAIN_ID || 1101
```

Comment:
It would be kind of nice to have a named constant for `1101` - the previous comments identifying which id is what was helpful.